pkgs/by-name/re/regina-normal/update.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  writeShellScript,
+  coreutils,
+  findutils,
+  curl,
+  gnupg,
+  gawk,
+  jq,
+  nix,
+  common-updater-scripts,
+
+  pname ? "regina-normal",
+  version,
+  url ? "https://api.github.com/repos/regina-normal/regina/releases/latest",
+  publicKey ? ./publickey.asc, # Benjamin Burton's PGP key (https://people.debian.org/~bab/regina-key.txt or https://keybase.io/benjaminburton/pgp_keys.asc)
```

```suggestion
  attrPath ? "regina-normal",
  url ? "https://api.github.com/repos/regina-normal/regina/releases/latest",
```
I think what matters here is that `regina-normal` is the attribute name for Regina in Nixpkgs.